packages/astro/test/server/sdk.test.ts

```diff
@@ -43,8 +43,6 @@ describe('Sentry server SDK', () => {
       expect(SentryNode.getIsolationScope().getScopeData().tags).toEqual({});
 
       init({ dsn: 'https://public@dsn.ingest.sentry.io/1337' });
-
-      expect(SentryNode.getIsolationScope().getScopeData().tags).toEqual({ runtime: 'node' });
```

Comment:
l: let's delete the entire test since this was the only relevant check. I don't think we need to assert that the isolation scope has no tags at the beginning for Astro specifically.